Contents/DefaultPrefs.json

```diff
@@ -88,6 +88,12 @@
     "type":"bool",
     "default":"false"
   },
+  {
+    "id":"collectionsfromtags",
+    "label":"Enable generating Collections from tags\n_____________________________________________________________________________________",
+    "type":"bool",
+    "default":"false"
```

Comment:
I would prefer the default being `true`, to keep the old behaviour. But then, how could we allow people to turn it off, and have collections that we created removed..?

Reply:
Good point. Changing the default behavior could be viewed as making this a breaking change...

I don't know much about Plex's back-end. This is the first time I have messed with it. I don't know how Plex handles removing collections internally, and to be honest that sort of action could back-fire. Someone somewhere might get upset if we removed a collection they wanted to keep.
It was annoying, but not the end of the world to manually clean up the extra collections that were generated.
If you want to make the default "true", then I would do so and just leave it at that. Perhaps some mention of this in the top-level instructions so folks can turn it off before running...?

Reply:
There regrettably are no top-level instructions other than how to install at the moment. I too would vote for the default being true, to keep the default behaviour. Everybody who wants to disable it, can do and recreating the library from scratch should clear all existing and the maybe unwanted collections from a first try with the default setting.

Reply:
Ok. When I get home tonight I'll change the default to be "true" and make a new commit.
Thanks for being open to this change. :)